subsys/net/lib/nrf_cloud/src/nrf_cloud_agps.c

```diff
@@ -69,19 +69,62 @@ static int type_array2str(enum gps_agps_type *types, size_t type_count,
 	return 0;
 }
 
-int nrf_cloud_agps_request(enum gps_agps_type *types, size_t type_count)
+int nrf_cloud_agps_request(const struct gps_agps_request request)
 {
 	int err, len;
 	char types_str[20];
 	char types_array[30];
-	char request[250];
+	char request_buf[250];
```

Comment:
This is a pretty large buffer to be put on the stack. @rlubos what's best practice in these cases? Make it static or leave it like this?

Reply:
It depends on the use-case really I guess. If we make it static, the function will no longer be reentrant (unless we add some kind of mutex protection).

Reply:
True that. It's fine by me to keep it as is.